python/hail/dataset.py

```diff
@@ -3043,10 +3043,11 @@ def linreg3(self, ys, covariates=[], root='va.linreg', use_dosages=False, varian
                       sparsity_threshold=numeric,
                       use_dosages=bool,
                       n_eigs=nullable(integral),
-                      dropped_variance_fraction=(nullable(float)))
+                      dropped_variance_fraction=nullable(float),
+                      filter_variants_expr=(nullable(strlike)))
```

Comment:
Yeah, I realize this has gotten long. Basically, when you submit a LD matrix, the method needs to multiply against the genotype matrix, so the VDS needs all the variants that are present in the LD matrix to perform the multiplication. However, we also don't want to require that they run the per variant association on all the variants. `filter_variants_expr` might be a bad name, but the point was to specify which of the variants in the VDS get per variant tested.